spec/dummy/spec/integration/user_commands_spec.rb

```diff
@@ -18,4 +18,13 @@
       expect(result.error.messages[:name]).to include("can't be blank")
     end
   end
+
+  describe 'delete' do
+    it 'deletes record' do
+      users.create.call(name: 'Piotr')
+      result = users.try { delete(:by_name, 'Piotr') }
+
+      expect(result.error).to be(nil)
```

Comment:
Few of them errored with https://travis-ci.org/rom-rb/rom-rails/jobs/45608784
Looks like they depend on order and assigned id from database.